src/esp/assets/Mp3dInstanceMeshData.cpp

```diff
@@ -55,7 +55,7 @@ bool Mp3dInstanceMeshData::loadMp3dPLY(const std::string& plyFile) {
     LOG(ERROR) << "Invalid element vertex header line";
     return false;
   }
-  int nVertex;
+  int nVertex = 0;
   iss >> token >> nVertex;
```

Comment:
... wait, why a custom PLY parser is still used here? :eyes:

Reply:
That's used just for processing the mp3d semantic ply's into what habitat ends up loading.  IIRC we left this here because it's used just fr that and the mp3d semantic ply's have a whole bunch of weird face attributes instead of our 1 weird one.